foxy/distribution.yaml

```diff
@@ -356,6 +356,21 @@ repositories:
       url: https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs.git
       version: master
     status: developed
+  avt_vimba_camera:
+    doc:
+      type: git
+      url: https://github.com/astuff/avt_vimba_camera.git
+      version: ros2_master
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/astuff/avt_vimba_camera-release.git
+      version: 2001.1.0-1
+    source:
+      type: git
+      url: https://github.com/astuff/avt_vimba_camera.git
```

Comment:
could you add a top level license file to the repo?

Reply:
Will do! this was originally forked from https://github.com/srv/avt_vimba_camera, which has no licence file, but the package.xml states "BSD". I'll create a BSD licence file if that sounds like the best approach.

Reply:
Sorry, I didn't notice you were releasing from a fork.
Do you have approval from the upstream maintainers to make a release?

In general, why prefer not allowing releases from forks.

Reply:
We were never able to make contact and assumed it was a dead project. See here: https://github.com/srv/avt_vimba_camera/issues/39

Reply:
I see.

That seems fine to me, I'll wait for an approval of a second reviewer in this case.
Maybe @sloretz or @hidmic (?).

Reply:
I'm not sure we have a policy for these cases. @clalancette @tfoote ?

Reply:
We do have a policy where we encourage people to contact the upstream and get the releases and changes done there.  However, if the original repository maintainers can't be contacted, we can release from a fork.

In this case, it does look like that was done, so :+1: from me on using this fork.